docs/develop/get-the-whole-document-from-an-add-in-for-powerpoint-or-word.md

```diff
@@ -41,33 +41,116 @@ Before you begin creating this add-in for PowerPoint or Word, you should be fami
-    </OfficeApp>
-    ```
+In a text editor, add the following code to the manifest file. If you're using a Visual Studio project, select the "Add-in only manifest" option.
+
+# [Unified manifest for Microsoft 365](#tab/jsonmanifest)
+
+> [!NOTE]
+> The unified manifest is generally available for production Outlook add-ins. It's available only for preview in Excel, PowerPoint, and Word add-ins.
+
+```json
+{
+    "$schema": "https://developer.microsoft.com/json-schemas/teams/vDevPreview/MicrosoftTeams.schema.json#",
+    "manifestVersion": "devPreview",
+    "version": "1.0.0.0",
+    "id": "[Replace_With_Your_GUID]",
+    "localizationInfo": {
+        "defaultLanguageTag": "en-us"
+    },
+    "developer": {
+        "name": "[Provider Name e.g., Contoso]",
+        "websiteUrl": "[Insert the URL for the app e.g., https://www.contoso.com]",
+        "privacyUrl": "[Insert the URL of a page that provides privacy information for the app e.g., https://www.contoso.com/privacy]",
+        "termsOfUseUrl": "[Insert the URL of a page that provides terms of use for the app e.g., https://www.contoso.com/servicesagreement]"
+    },
+    "name": {
+        "short": "Get Doc add-in",
+        "full": "Get Doc add-in"
+    },
+    "description": {
+        "short": "My get PowerPoint or Word document add-in.",
+        "full": "My get PowerPoint or Word document add-in."
+    },
+    "icons": {
+        "outline": "_layouts/images/general/office_logo.jpg",
+        "color": "_layouts/images/general/office_logo.jpg"
+    },
+    "accentColor": "#230201",
+    "validDomains": [
+        "https://www.contoso.com"
+    ],
+    "showLoadingIndicator": false,
+    "isFullScreen": false,
+    "defaultBlockUntilAdminAction": false,
+    "authorization": {
+        "permissions": {
+            "resourceSpecific": [
+                {
+                    "name": "Document.ReadWrite.User",
+                    "type": "Delegated"
+                }
+            ]
+        }
+    },
+    "extensions": [
+        {
+            "requirements": {
+                "scopes": [
+                    "document",
+                    "presentation"
+                ]
+            },
+            "alternates": [
+                {
+                    "alternateIcons": {
+                        "icon": {
+                            "size": 32,
+                            "url": "http://officeimg.vo.msecnd.net/_layouts/images/general/office_logo.jpg"
+                        },
+                        "highResolutionIcon": {
+                            "size": 64,
+                            "url": "http://officeimg.vo.msecnd.net/_layouts/images/general/office_logo.jpg"
+                        }
+                    }
+                }
+            ]
+        }
+    ]
+}
+```
 
-1. Save the file as **GetDoc_App.xml** using UTF-8 encoding to a network location or to an add-in catalog.
+Save the file as **GetDoc_App.json** to a network location or to an add-in catalog.
```

Comment:
This won't work. A JSON manifest and 2 image files get added to a zip file and it is the zip file that is sideloaded. This article doesn't say much about sideloading anyway, so I think we can just delete this sentence.